### PR TITLE
Added resource url function + tests

### PR DESCRIPTION
**Pull request by AaronLaBrie**

### CODE REVIEW

**Comment by AaronLaBrie** [comment]
Waiting on an internal change to merge this.

File: README.md [hunk 1]
```diff
@@ -101,6 +101,14 @@ If an endpoint supports both query params and a post body, you can pass the quer
 .post({email:"test@test.com"}, {foo: "bar"})
 ```
 
+Resource URLs
+----------------------------------
+Certain endpoints, such as `/2.0/accounts/threads` will return a complete URL that you can call to access a resource. You can use the `resource()` function to call these urls. Parameters are passed as normal.
+
+```js
+ctxioClient.resource(resource_url).get().then(function() {...})
```
Comment (tantona) [inline README.md]: is the response passed to the `.then(function(response) {})` ?

**Comment by tantona** [comment]
@AaronLaBrie do we raise any errors if a bad url is passed or if the request fails?  I suppose this will bubble up and the dev will be able to `.catch()` them, but it might be nice to know where the error occured

File: README.md (hunk 1 above)
Comment (AaronLaBrie) [inline README.md]: Yes, I'll update that line

**Comment by AaronLaBrie** [comment]
@tantona It'll just hit the api and throw back whatever it does. This should only be used for resoure_url's returned from our calls, so there shouldn't be issues there. 

If people want to do their own thing with it, and it breaks, that's their problem.

**Comment by tantona** [comment]
:+1:

**Pull request merged**